crates/meroctl/src/cli/identity/generate.rs

```diff
@@ -12,8 +13,12 @@ pub struct GenerateCommand;
 
 impl Report for GenerateContextIdentityResponse {
     fn report(&self) {
-        println!("public_key: {}", self.data.public_key);
-        println!("private_key: {}", self.data.private_key);
+        println!("{} {}", "✓".green(), "Key pair generated".bold());
+        println!("  Public Key: {}", self.data.public_key.to_string().cyan());
+        println!(
+            "  Private Key: {}",
+            self.data.private_key.to_string().cyan()
```

Comment:
`OwoColorize` is [implemented](https://docs.rs/owo-colors/latest/owo_colors/trait.OwoColorize.html#impl-OwoColorize-for-D) for any generic `D` unconditionally

and return types of its methods are `Display` if `D: Display` ([example](https://docs.rs/owo-colors/latest/owo_colors/struct.FgColorDisplay.html#impl-Display-for-FgColorDisplay%3C'a,+Color,+T%3E))

so if you can call `.to_string()` before calling any `OwoColorize` method like `.cyan()`, then you never need to call `.to_string()`